server/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const port = 4001;
+
+const router = require('./router');
+
+app.use(cors());
+app.use(express.json());
+app.use(router);
+
+app.listen(port, () => {
+  console.log(`Listening on port ${port}!`) //eslint-disable-line no-console
+})
```

Comment:
if you want you can add a logger as first middleware...there is a npm package and it will log for you in the terminal all incoming requests. Not a must but can be very helpful in certain occasions.